libnavigation-core/src/main/java/com/mapbox/navigation/core/routeoptions/RouteOptionsUpdater.kt

```diff
@@ -30,10 +31,12 @@ class RouteOptionsUpdater {
      * @return `RouteOptionsResult.Error` if a new [RouteOptions] instance cannot be combined based on the input given.
      * `RouteOptionsResult.Success` with a new [RouteOptions] instance if successfully combined.
      */
+    @JvmOverloads
     fun update(
         routeOptions: RouteOptions?,
         routeProgress: RouteProgress?,
         locationMatcherResult: LocationMatcherResult?,
+        rerouteOptions: RerouteOptions? = null,
```

Comment:
```suggestion
        rerouteOptions: RerouteOptions = RerouteOptions.Builder().build(),
```

Reply:
I moved `rereouteOptions` work to `RererouteController`: `RouteOptionsUpdate` might be used by any, not only RerouteController, and looks like providing RerouteOptions here is not a good option